frontend/public/style/_overrides.scss

```diff
@@ -21,6 +21,13 @@ form.pf-c-form {
   }
 }
 
+.co-m-pane__body .pf-c-toolbar {
```

Comment:
How many places are these overrides needed? We have a common toolbar component for list page tables at least.

I'd recommend creating a specific `co-toolbar` class or similar that applies these overrides to only the toolbars we need.